packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -102,7 +111,24 @@ class AmountSelectors extends React.Component {
   };
 
   render() {
-    const { disabled } = this.props;
+    const {
+      disabled,
+      max,
+      min,
+      minAmountMessage,
+      maxAmountMessage,
+      showBoundariesMessageOnlyOnFocus,
+    } = this.props;
+    const { focused, value } = this.state;
+    const showMessage =
+      !showBoundariesMessageOnlyOnFocus ||
+      (showBoundariesMessageOnlyOnFocus && focused);
+    const showMaxMessage = value > max && showMessage;
+    const showMinMessage = value < min && showMessage;
```

Comment:
If I enter a number then a point, the buttons become disabled and show `minAmountMessage `
![Screen Shot 2019-09-27 at 11 23 43](https://user-images.githubusercontent.com/32174637/65776852-49cb4100-e119-11e9-934d-d835eb2cbc9a.png)